scripts/download_pix3d.py

```diff
@@ -0,0 +1,35 @@
+from sys import version_info
+
+if version_info.major == 2:
+    raise Exception("This script only works with python3.x!")
+
+import os
+from urllib.request import urlretrieve, build_opener, install_opener
+import subprocess
+
+
+if __name__ == "__main__":
+    # setting the default header, else the server does not allow the download
+    opener = build_opener()
+    opener.addheaders = [('User-agent', 'Mozilla/5.0')]
+    install_opener(opener)
+
+    # set the download directory relative to this one
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    pix3d_dir = os.path.join(current_dir, "..", "resources", "pix3d")
+
+    if not os.path.exists(pix3d_dir):
+        os.makedirs(pix3d_dir)
+
+    # download the zip file, which contains all the obj files
+    print("Download the zip file, may take a while:")
```

Comment:
Could we include the size of the file here, so that the user can better estimate how long this will take.